content/docs/operating/security.md

```diff
@@ -177,6 +186,33 @@ and bandwidth.
 It is recommended to monitor all components for failure, and to have them
 automatically restart on failure.
 
+## Server-side TLS and authentication
+
+In the future, server-side TLS support will be rolled out to the different
+projects under the Prometheus organization. Those projects include Prometheus,
+Alertmanager, Pushgateway and the official exporters.
+
+The Go projects will share the same TLS library, which will be based on the
+Go vanilla TLS library. We keep [Go default TLS
+parameters](https://golang.org/pkg/crypto/tls/#Config), with one exception: we
+support only TLSv1.2 and higher.
+
+TLS will be added to Java exporters in the future.
+
+If you have special TLS needs, like a different cipher suite or older TLS
+version, the TLS support on the projects should enable you to build a secure
+tunnel between the servers and reverse proxies with special settings.
+
+HTTP Basic Authentication is also supported. In such a case, usernames
+and passwords are provided in the same configuration file as the TLS
```

Comment:
Hmm, this is the first mention of a configuration file which makes it seem as though we forgot that - though I don't think we need this level of detail in this doc (that's more for the usage docs).

Reply:
that config file has security implications -- if people give access to anyone it is possible to change certs, disable client side auth, add users..

Reply:
What goes in what config files is too detailed to go here I think, plus other config files also have security implications depending on your setup.